Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN chmod +x /opt/wait-for-it.sh
 COPY ./startup.relational.dev.sh /opt/startup.relational.dev.sh
 RUN chmod +x /opt/startup.relational.dev.sh
 RUN sed -i 's/\r//g' /opt/wait-for-it.sh
-RUN sed -i 's/\r//g' /opt/startup.relational.dev.sh
+# RUN sed -i 's/\r//g' /opt/startup.relational.dev.sh
```

Comment:
Why are we commenting this out?

Reply:
The command formatted the bash file to produce an error, commented out temporarily to test docker setup.